### PR TITLE
component scan from different packages

### PR DESCRIPTION
**Pull request by RichardW98**
java -jar -Dspring.profiles.active=local **CONFIGURATION_PACKAGE=org.parodos.*** target/workflow-service-1.0.3-SNAPSHOT.jar

custom workflow is in package org.parodos.workflow.examples

Screenshot 2023-03-21 at 5 59 27 PM

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
looks good to me

**Comment by masayag** [comment]
Do you think we should add a section on it to the workflow-service/README.md?

**Comment by masayag** [comment]
lgtm

File: workflow-service/README.md [hunk 1]
```diff
@@ -136,3 +136,32 @@ Parodos
 
 Yes. In this first release a configuration pattern widely used across many enterprise environments was chosen. However,
 future release will include a DSL (domain specific language) for configuring Workflows without have to write Java code.
+
+### How to configure my postgres database and my workflow configuration class in different package?
+
+before starting this workflow-service application, set the following environment variables for the database: ,
```
Comment (masayag) [inline workflow-service/README.md:142]: pls remove the comma at the end of the line

File: workflow-service/README.md [hunk 2]
```diff
@@ -136,3 +136,32 @@ Parodos
...
+* CONFIGURATION_PACKAGE
+
+example: 
+```
+export CONFIGURATION_PACKAGE=org.test.*
+```
+
+for multiple packages:
+```
+export CONFIGURATION_PACKAGE=org.test.*, dev.test2.*
```
[the remark this answers is not in the document]
Comment (RichardW98) [reply workflow-service/README.md]: fixed

**Comment by pkliczewski** [comment]
@RichardW98 please rebase

**Comment by pkliczewski** [comment]
@RichardW98 It sees like it still cannot be rebased safely. Please rebase

**Pull request merged**